packages/app/src/app/pages/Dashboard/Components/Repository/RepositoryListItem.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {
+  Element,
+  Column,
+  Grid,
+  Icon,
+  IconButton,
+  ListAction,
+  Stack,
+  Text,
+} from '@codesandbox/components';
+import { css } from '@styled-system/css';
+import { RepositoryProps } from './types';
+
+export const RepositoryListItem: React.FC<RepositoryProps> = ({
+  labels,
+  repository,
+  onContextMenu,
+  ...props
+}) => {
+  return (
+    <ListAction
+      align="center"
+      css={css({
+        paddingX: 0,
+        height: 64,
+        borderBottom: '1px solid',
+        borderBottomColor: 'grays.600',
+        overflow: 'hidden',
+        backgroundColor: 'transparent',
+        color: 'inherit',
+        ':hover, :focus, :focus-within': {
+          backgroundColor: 'list.hoverBackground',
```

Comment:
this might conflict with the changes @lenasmayor did on #6942

Reply:
@alexnm thanks for the heads-up, merged #6942 and rebased this branch.